### PR TITLE
Environment usage refactor

### PR DESCRIPTION
**Pull request by xsalonx**

### CODE REVIEW

File: app/lib/ResProvider.js [hunk 1]
```diff
@@ -215,6 +216,26 @@ class ResProvider {
         }
         return process.env.ALIMONITOR_PASSPHRASE;
     }
+
+    static endpointFor(serviceAbbr) {
```
Comment (Ehevi) [inline app/lib/ResProvider.js:220]: Why not simply `getServiceEndpoint`?

Comment (xsalonx) [reply app/lib/ResProvider.js]: done

File: app/lib/ResProvider.js [hunk 2]
```diff
@@ -215,6 +216,26 @@ class ResProvider {
...
+        const varsDef = {};
+        varsDef[`RCT_EP_${serviceAbbr}_PROT`] = 'prot';
+        varsDef[`RCT_EP_${serviceAbbr}_HOST`] = 'host';
+        varsDef[`RCT_EP_${serviceAbbr}_PORT`] = 'port';
+        varsDef[`RCT_EP_${serviceAbbr}_PATH`] = 'path';
+        const p = ResProvider.viaEnvVars(varsDef, null, 'warn');
+        const { host } = p;
+        let { path } = p;
+        // eslint-disable-next-line prefer-destructuring
+        let prot = p['prot'];
```
Comment (Ehevi) [inline app/lib/ResProvider.js:231]: Have you tried the following?
```
let { prot } = p;
```

File: app/lib/ResProvider.js [hunk 3]
```diff
@@ -215,6 +216,26 @@ class ResProvider {
...
+        prot = prot ? prot : 'https';
+        // eslint-disable-next-line prefer-destructuring
+        let port = p['port'];
```
Comment (Ehevi) [inline app/lib/ResProvider.js:234]:
```
let { port } = p;
```

**Review by Ehevi — APPROVED** [review]
Plese give my comments a try before merging. Apart from that, everything looks good.

File: app/lib/ResProvider.js (hunk 3 above)
Comment (xsalonx) [reply app/lib/ResProvider.js:234]: Port and prot might be not defined

**Pull request merged**